tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -461,6 +467,12 @@ dualtor/test_orchagent_mac_move.py:
     conditions:
       - "(topo_type not in ['t0']) or ('dualtor' in topo_name)"
 
+dualtor/test_orchagent_slb.py:
+  skip:
+    reason: "KVM do not support dualtor tunnel functionality, lower tor bgp verify would fail. Temporarily skip in PR testing"
```

Comment:
We can remove `Temporarily skip in PR testing` as it's expected behavior and no plan to repair